[BACKPORT] STM32F7 and STM32H7 SDMMC internal pull up usage fixed

davids5 opened the pull request:
Code was flawed in that the Pins are defined with the
    pullups in the definition. Since there are no alterntes pins
    there is no way to remove them. So not enabling the CONFIG
    pull up did nothing as did enabling them.

    Code also ignored the use of D0 for ready detection causing
    3X+ the chatter.

    This is now a compile time feature as there was no reason for
    it to be a run time. It wasted both flash and ram.

NaterGator commented on nuttx/drivers/usbdev/cdcacm.c:
```diff
@@ -1963,38 +1967,63 @@ static int cdcuart_ioctl(FAR struct file *filep,int cmd,unsigned long arg)
-  return ret;
-}
+        return ret;
+      }
```
Is the extra whitespace here intentional?

davids5 replied:
No  C&P error - Thanks

 

From: Nate Weibley [mailto:notifications@github.com] 
Sent: Saturday, August 20, 2016 6:12 AM
To: PX4/NuttX
Cc: David Sidrane; Author
Subject: Re: [PX4/NuttX] Backport of Serial and CDCACM fixes from upstream (#78)

 

In nuttx/drivers/usbdev/cdcacm.c <https://github.com/PX4/NuttX/pull/78#discussion_r75580973> :

>  
> -  return ret;
> -}
> +        return ret;
> +      }

Is the extra whitespace here intentional? 

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub <https://github.com/PX4/NuttX/pull/78/files/e7d315ab22cb96733f49d9d1a7e1e1e726e48150#r75580973> , or mute the thread <https://github.com/notifications/unsubscribe-auth/AB2w3RjTWaTpwWI70aYrSgcQZ5b5ZdRBks5qhydPgaJpZM4JollL> .  <https://github.com/notifications/beacon/AB2w3TTcViEtagfb6ZdtO9FdOvnE-o70ks5qhydPgaJpZM4JollL.gif>

davids5 commented:
Closed in favor of https://github.com/PX4/NuttX/pull/80 as a work around.

davids5 committed:
[BACKPORT] STM32F7 and STM32H7 SDMMC internal pull up usage fixed

    Code was flawed in that the Pins are defined with the
    pullups in the definition. Since there are no alterntes pins
    there is no way to remove them. So not enabling the CONFIG
    pull up did nothing as did enabling them.

    Code also ignored the use of D0 for ready detection causing
    3X+ the chatter.

    This is now a compile time feature as there was no reason for
    it to be a run time. It wasted both flash and ram.
